index.html

```diff
@@ -1,11 +1,167 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Homework 2</title>
 </head>
-<body>
-  
+
+<body class="page__body">
+  <header>
+    <a class="logo" href="index.html">
+      <span class="logo__web">Web</span>
+      <span class="logo__studio logo__studio--black">Studio</span>
+    </a>
+    <ul class="navigation">
+      <li>
+        <a class="navigation__link" href="#">Студія</a>
+      </li>
+      <li>
+        <a class="navigation__link" href="portfolio.html">Портфоліо</a>
+      </li>
+      <li>
+        <a class="navigation__link" href="#">Контакти</a>
+      </li>
+    </ul>
```

Comment:
This navigation list should be wrapped in a `<nav>` tag with a classname `nav`, `ul` should have a class name of `nav__list`, `li` should have a class name of `nav__item`, and `a` may have a class `nav__link`

```suggestion
  <nav class="nav">
    <ul class="nav__list">
      <li class="nav__item">
        <a class="nav__link" href="#">Студія</a>
      </li>
      <li class="nav__item">
        <a class="navigation__link" href="portfolio.html">Портфоліо</a>
      </li>
      <li class="nav__item">
        <a class="navigation__link" href="#">Контакти</a>
      </li>
    </ul>
```